generators/app/templates/infra/gcp/.tflint.hcl

```diff
@@ -0,0 +1,9 @@
+plugin "google" {
+    enabled = true
+    version = "0.21.2"
```

Comment:
This version does not exist. Latest is "0.24.0" (and same for aws, we can update it)

Reply:
And with latest version you will need a `main.tf` file